.gitignore

```diff
@@ -0,0 +1,2 @@
+gen
```

Comment:
I usually add a `/` at the end here to show that they are folders. Not sure what's preferred.